<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCallData</class>
 <widget class="QWidget" name="VideoCallData">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>528</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#VideoCallData{
	border:1px solid #E5E8EF;
}

QWidget{
background:#FFFFFF;
}

*{
	font-family : 'Microsoft YaHei';
	font-size:12px;
    color:#4E5969;
	outline:0px;
}

#content_widget{
background:#FFFFFF;
}

#lbl_title{
	font-size: 14px;
	color: #1D2129;
}
#btn_cancel{
background: #F2F3F8;
border-radius: 2px;
}

#btn_confirm{
	color:#fff;
	background: #1664FF;
	border-radius: 2px;
}

QScrollArea
{
border:none;
}

QScrollBar:vertical 
{
border: none;
background:#FFFFFF;
width: 4px;
margin: 0px;
}
QScrollBar::handle:vertical { 
	background:#C9C9C9; 
    min-height: 20px;
	width: 4px;
	border-radius:2px;
}
QScrollBar::add-line:vertical {background-color: 
#FFFFFF;height: 4px;}
QScrollBar::sub-line:vertical {background-color: 
#FFFFFF;height: 4px;}
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical 
{height: 0px;}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{background: #FFFFFF;}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="top_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>46</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>301</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_title">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>real-time_data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>301</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_close{border: none;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/video_call_resource.qrc">
          <normaloff>:/img/videocall_close</normaloff>:/img/videocall_close</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="option" native="true">
     <property name="styleSheet">
      <string notr="true">#option{
	border-top:1px solid #E5E8EF;
}

QRadioButton::indicator{
width: 12px;
height: 12px; 
border-radius: 6px;
}
QRadioButton::indicator:checked{
image: url(:/img/videocall_radio_button_checked);
}
QRadioButton::indicator:unchecked{
image: url(:/img/videocall_radio_button_unchecked);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QRadioButton" name="videoButton">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>video</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="audioButton">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>audio</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="content_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>362</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="confirmWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#confirmWidget{
	border-top:1px solid #E5E8EF;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_confirm">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/video_call_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>VideoCallData</receiver>
   <slot>onClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>956</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_confirm</sender>
   <signal>clicked()</signal>
   <receiver>VideoCallData</receiver>
   <slot>onConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>VideoCallData</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClose()</slot>
  <slot>onConfirm()</slot>
  <slot>onCancel()</slot>
 </slots>
</ui>
